tools/iothrottle_example.txt

```diff
@@ -0,0 +1,60 @@
+Demonstrations of iothrottle, the Linux eBPF/bcc version.
+
+iothrottle traces the buffer io throttle's latency system-wide, and prints variout details.
+Example output:
+
+# ./iothrottle
+COMM  PID    TID    FNAME             COUNT  POS          BDI    DIRTIED  RATELIMIT(KBps)    PAUSED(ms)      LAT(ms)
+bio   5244   5244   bio_rrw_test.txt  32768  2427650048   259:3  8        3376                       13        13.58
+fio   5243   5243   fio_rrw_test.txt  32768  1802076160   259:3  8        6052                       13        14.67
+bio   5244   5244   bio_rrw_test.txt  32768  2427748352   259:3  8        6100                       13        13.07
+fio   5243   5243   fio_rrw_test.txt  32768  1802174464   259:3  8        6100                       13        13.84
+bio   5244   5244   bio_rrw_test.txt  32768  2427813888   259:3  8        5112                       13        13.44
+fio   5243   5243   fio_rrw_test.txt  32768  1802240000   259:3  8        2720                       13        13.81
+bio   5244   5244   bio_rrw_test.txt  32768  2427879424   259:3  8        2720                       14        13.86
+fio   5243   5243   fio_rrw_test.txt  32768  1802272768   259:3  8        2720                       14        15.24
+bio   5244   5244   bio_rrw_test.txt  32768  2427912192   259:3  8        2720                       14        14.03
+fio   5243   5243   fio_rrw_test.txt  32768  1802305536   259:3  8        2860                       14        15.94
+^C
+
+While tracing, the processes write buffer IO, when there are too many dirty pages, balance_dirty_pages will be called
+to wake up bdi writeback thread and give up cpu by io_schedule_timeout(paused), which will increase the latency of
+vfs_write.
+
+iothrottle can be useful for discovering this situation and give some judge:
+- the buffer IO's performance jitter
+- the bdi's performance
+- whether we should make /proc/sys/vm/dirty_background_ratio lower
```

Comment:
Based on your above data, could you give more explanation on the above three listed items? For example, what data indicates buffer io performance jitter, what is the bdi's performance and in this particular case, should we make /proc/sys/vm/dirty_background_ratio lower or not and why.